app/src/main/java/com/antosik/benchproject/app/more/view/MoreFragment.kt

```diff
@@ -18,8 +47,134 @@ class MoreFragment : NavigableFragment<MoreFragmentNavigator>() {
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
-    ) = MoreFragmentBinding.inflate(inflater, container, false).apply {
-        lifecycleOwner = viewLifecycleOwner
-        viewModel = moreViewModel
-    }.root
+    ) = ComposeView(requireContext()).apply {
```

Comment:
Would be nice to add `Preview`

Reply:
https://developer.android.com/jetpack/compose/tooling/previews

Reply:
I added it :D